.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,20 @@
-## Thank you!
+## Thank you for your contribution!
 
-Thank you for your contribution.
-Please make sure that your submission includes the following:
+We appreciate the time and effort you've put into this pull request.
+To help us review it efficiently, please ensure you've gone through the following checklist:
 
-### Must
 
-- [ ] The code compiles without `errors` or `warnings`.
-- [ ] All examples work.
-- [ ] `cargo fmt` was run.
-- [ ] Your changes were added to the `CHANGELOG.md` in the proper section.
-- [ ] You updated existing examples or added examples (if applicable).
-- [ ] Added examples are checked in CI
+### Submission Checklist 📝
+- [ ] Examples related to your contribution work as expected
```

Comment:
Maybe "Examples related to your contribution work as expected" and "I have updated existing examples or added examples (if applicable)." is a bit redundant?

Reply:
Yes, maybe we should go with only one of them. 
"I have updated existing examples or added examples (if applicable)" seems to "cover" most of wanted PR characteristic 🤔